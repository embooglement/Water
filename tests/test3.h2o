let x = 1;
let y = 2;
{
	let z = x + y;
	println(x, y, z);
}
{
	var z = x + y;
	println(x, y, z);
}

let f = func() {
	var x = 3;
	let z = x + y;
	println(x, y, z);
};

let g = func() {
	var z = x + y;
	println(x, y, z);
};

let h = func(x, y) {
	let z = x + y;
	println(x, y, z);
};

f();
g();
h(8, 9);
