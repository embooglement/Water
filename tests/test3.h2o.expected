1 2 3
1 2 3
3 2 5
1 2 3
8 9 17

